stoplight.js

```diff
@@ -0,0 +1,22 @@
+//create variable for all bulbs
+var $bulbs = $('.bulb');
+var $yellow = $('#yellow');
+var $red = $('#red');
+
+//register event listener
+$bulbs.on('click', function() {
+  activate($(this));
+})
+
+$yellow.on('click', function(){
+  window.setTimeout(function() {
+    activate($red);
```

Comment:
Everything works, but I noticed that the yellow light fades in and out twice when you click on it...